src/main/java/org/elastos/hive/vendors/onedrive/OneDriveAuthHelper.java

```diff
@@ -164,15 +168,18 @@ public void cancelled() {}
 		public void completed(HttpResponse<JsonNode> response) {
 			if (response.getStatus() != 200) {
 				HiveException ex = new HiveException(response.getStatusText());
-				this.callback.onError(ex);
+				if (this.callback != null) {
```

Comment:
Same as above,  don't have to check this.callback is null.

Reply:
Ok, i'll add the codes to check.